recipes/cuda-nvrtc/meta.yaml

```diff
@@ -0,0 +1,154 @@
+{% set name = "cuda-nvrtc" %}
+{% set version = "12.0.76" %}
+{% set cuda_version = "12.0" %}
+{% set platform = "linux-x86_64" %}  # [linux64]
+{% set platform = "linux-ppc64le" %}  # [ppc64le]
+{% set platform = "linux-sbsa" %}  # [aarch64]
+{% set platform = "windows-x86_64" %}  # [win]
+{% set target_name = "x86_64-linux" %}  # [linux64]
+{% set target_name = "ppc64le-linux" %}  # [ppc64le]
+{% set target_name = "sbsa-linux" %}  # [aarch64]
+{% set target_name = "x64" %}  # [win]
+{% set extension = "tar.xz" %}  # [not win]
+{% set extension = "zip" %}  # [win]
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://developer.download.nvidia.com/compute/cuda/redist/cuda_nvrtc/{{ platform }}/cuda_nvrtc-{{ platform }}-{{ version }}-archive.{{ extension }}
+  sha256: 0a4ebc9a1516a5e00f14c69365ba782dcfab545d2abb15740569970f89855bff  # [linux64]
+  sha256: a8a5ad2c6823435762acb209fbfb032d22fc3d09c4bfc0cade8d3734f4a26c60  # [ppc64le]
+  sha256: 40d88b96fd41b28f2c5a14fa14a2c16559b0abb77689d1f85525789b23e21cad  # [aarch64]
+  sha256: 2e97727ea5c943d0757e564fe87818a5c8c3d27bc78d60a7aaab8874e3ec5960  # [win]
+
+build:
+  number: 0
+  skip: true  # [osx]
+
+outputs:
+  - name: cuda-nvrtc
+    files:
+      - lib/libnvrtc*.so.*                            # [linux]
+      - targets/{{ target_name }}/lib/libnvrtc*.so.*  # [linux]
+      - Library\bin\nvrtc*.dll                        # [win]
+    requirements:
+      build:
+        - {{ compiler("c") }}
+        - {{ compiler("cxx") }}
+        - arm-variant * {{ arm_variant_type }}  # [aarch64]
+        - sysroot_{{ target_platform }} 2.17    # [linux]
+      host:
+         - cuda-version {{ cuda_version }}
+      run:
+         - {{ pin_compatible("cuda-version", max_pin="x.x") }}
+    test:
```

Comment:
Please merge this PR to fix tests: https://github.com/adibbley/staged-recipes/pull/5